corehq/apps/toggle_ui/views.py

```diff
@@ -30,10 +31,36 @@ def page_url(self):
         return reverse(self.urlname)
 
     @property
+    def show_usage(self):
```

Comment:
mmm...I see your point, but I'm going to say the action is "show" and leave it as is :-p